test/unit/org/apache/cassandra/cql3/functions/masking/ColumnMaskTest.java

```diff
@@ -207,22 +207,22 @@ public void testColumnMaskingWithNotNativeFunction() throws Throwable
     }
 
     @Test
+    @SuppressWarnings("resource")
```

Comment:
What's the point of of replacing the auto-closable by this?

Reply:
The sessions provided by `CQLTester#sessionNet()` are cached in `CQLTester.sessions`. If we close the session (manually or with try-with-resources) then the next call to `CQLTester#sessionNet()` will receive a closed session. 

Note that the multiple direct uses of `CQLTester#sessionNet()` on other tests never try to close the session, and all the flavours of `CQLTester#sessionNet` don't close it either. That seems to indicate that not closing the reusable session is the intended use, even if the warning suppression is not commonly used.

The sessions are later closed in `CQLTester.tearDownClass`.

Reply:
Thx!